### PR TITLE
Text Analytics perf tests

### PR DESCRIPTION
**Pull request by srnagar**
This PR adds perf test maven module for Text Analytics track 2 client library. The test uses a local http client to run the perf tests as the real Azure service throttles requests and cannot effectively test the performance of the client library. Currently, the perf tests are only for language detection API only. Perf tests for other APIs will be done in separate PRs.

### CODE REVIEW

**Comment by mssfang** [comment]
@srnagar Seems we have duplicate the works for TA performance tests. https://github.com/Azure/azure-sdk-for-java/pull/20196

File: eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml [hunk 1]
```diff
@@ -399,12 +399,12 @@ the main ServiceBusClientBuilder. -->
             files="com.azure.cosmos.ClientSideRequestStatistics"/> <!-- Need OperatingSystemMXBean from sun to obtain cpu info -->
 
   <!-- perf-test files are not public API, so they don't need to fulfill all requirements. -->
-  <suppress checks="Javadoc" files=".*[/\\]azure-storage-perf|azure-search-perf|azure-identity-perf|azure-template-perf[/\\].*\.java"/>
+  <suppress checks="Javadoc" files=".*[/\\]azure-storage-perf|azure-search-perf|azure-identity-perf|azure-ai-textanalytics-perf|azure-template-perf[/\\].*\.java"/>
```
Comment (samvaity) [inline eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml:402]: I think according to latest guidance its suggested to have javadocs for perf tesing libs.

File: sdk/textanalytics/azure-ai-textanalytics-perf/src/main/java/com/azure/ai/textanalytics/perf/ServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    protected final TextAnalyticsClient textAnalyticsClient;
+
+    /**
+     * Creates an instance of performance test.
+     * @param options the options configured for the test.
+     */
+    public ServiceTest(TextAnalyticsStressOptions options) {
+        super(options);
+        textAnalyticsAsyncClient = new TextAnalyticsClientBuilder()
+            .endpoint(Configuration.getGlobalConfiguration().get("AZURE_TA_ENDPOINT"))
```
Comment (samvaity) [inline sdk/textanalytics/azure-ai-textanalytics-perf/src/main/java/com/azure/ai/textanalytics/perf/ServiceTest.java:28]: Should we mention about the required creds to run this test in the readme setup document?

File: sdk/textanalytics/azure-ai-textanalytics-perf/src/main/java/com/azure/ai/textanalytics/perf/ServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            .credential(new AzureKeyCredential(Configuration.getGlobalConfiguration().get("AZURE_TA_ACCESS_KEY")))
+            .httpClient(new LocalHttpClient(options.getDocumentCount()))
+            .buildAsyncClient();
+
+        textAnalyticsClient = new TextAnalyticsClientBuilder()
+            .endpoint(Configuration.getGlobalConfiguration().get("AZURE_TA_ENDPOINT"))
```
Comment (samvaity) [inline sdk/textanalytics/azure-ai-textanalytics-perf/src/main/java/com/azure/ai/textanalytics/perf/ServiceTest.java:34]: We should throw an error if its not able to get this configuration?

File: sdk/textanalytics/azure-ai-textanalytics-perf/src/main/java/com/azure/ai/textanalytics/perf/ServiceTest.java (hunk 2 above)
Comment (samvaity) [reply sdk/textanalytics/azure-ai-textanalytics-perf/src/main/java/com/azure/ai/textanalytics/perf/ServiceTest.java:28]: nit: Should we use TA abbreviation?

**Comment by srnagar** [comment]
Closing this PR in favor of https://github.com/Azure/azure-sdk-for-java/pull/20196

**Pull request closed without merging**